modules/transports/core/nhttp/src/main/resources/stream-interceptors.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+  ~ Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<!--resides in conf folder-->
+<interceptors>
+    <interceptor name="SampleStreamInterceptor" class="org.apache.synapse.transport.passthru.SampleStreamInterceptor">
+        <parameter name="charset" value="ISO-8859-1"/>
+        <parameter name="enableInterception" value="true"/>
```

Comment:
Why we need a global config to enable? And do we enable the sample interceptor by default?

Reply:
just a sample implementation, user can refer this. This is not added to the conf.